src/sourcify-chains.ts

```diff
@@ -793,6 +793,14 @@ const sourcifyChainsExtensions: SourcifyChainsExtensionsObject = {
     contractFetchAddress: "https://goerli.basescan.org/" + ETHERSCAN_SUFFIX,
     txRegex: ETHERSCAN_REGEX,
   },
+  "8453": {
+    // Base Mainnet
+    supported: true,
+    monitored: false,
+    contractFetchAddress: "https://basescan.org/" + ETHERSCAN_SUFFIX,
+    rpc: ["https://developer-access-mainnet.base.org"],
```

Comment:
can remove this line once `chains.json` is regenerated (now that https://github.com/ethereum-lists/chains/pull/3165 is merged)